app/Http/Requests/Album/MergeAlbumsRequest.php

```diff
@@ -29,6 +30,13 @@ class MergeAlbumsRequest extends BaseApiRequest implements HasAlbum, HasAlbums
 	use HasAlbumsTrait;
 	use AuthorizeCanEditAlbumAlbumsTrait;
 
+	public function __construct(array $query = [], array $request = [], array $attributes = [], array $cookies = [], array $files = [], array $server = [], $content = null)
+	{
+		parent::__construct($query, $request, $attributes, $cookies, $files, $server, $content);
+
+		$this->albums = new Collection(); // initialize with empty collection to avoid error MergeAlbumsRequest::$albums must not be accessed before initialization
```

Comment:
I'm not sure if i am doing this correctly, I was getting an error 
> MergeAlbumsRequest::$albums must not be accessed before initialization

when i comment this line and run MerglbRequestTest